Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/DiagonalVelocityDampingForceField.h

```diff
@@ -44,12 +44,22 @@ class DiagonalVelocityDampingForceField : public core::behavior::ForceField<Data
     typedef core::objectmodel::Data<VecCoord> DataVecCoord;
     typedef core::objectmodel::Data<VecDeriv> DataVecDeriv;
 
-    /// air drag coefficient.
-    Data< VecDeriv > dampingCoefficients;
+    union
+    {
+        /// velocity damping coefficients (by cinematic dof)
+        Data< VecDeriv > d_dampingCoefficients;
+
+        SOFA_ATTRIBUTE_DEPRECATED("v23.12", "v24.06", "This Data is now replaced by d_dampingCoefficients")
+        Data< VecDeriv > dampingCoefficients;
+    };
 
 protected:
 
     DiagonalVelocityDampingForceField();
+    ~DiagonalVelocityDampingForceField()
+    {
+        d_dampingCoefficients.~Data();
```

Comment:
What is this line for ?

Reply:
https://github.com/sofa-framework/sofa/pull/4005#issuecomment-1639693734